spec/acceptance/agent_and_server_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper_acceptance'
+require 'simp/beaker_helpers'
+
+include Simp::BeakerHelpers
+
+describe 'hyperglass::server class' do
+  # https://github.com/simp/rubygem-simp-beaker-helpers#run_fake_pki_ca_on
+  # rubocop:disable RSpec/EmptyExampleGroup
+  context 'it setups a fake CA' do
+    Dir.mktmpdir do |cert_dir|
+      run_fake_pki_ca_on(hosts.first, hosts, cert_dir)
+      hosts.each { |sut| copy_pki_to(sut, cert_dir, '/etc/puppetlabs/puppet/ssl/') }
+    end
+    shell('mkdir -p /etc/puppetlabs/puppet/ssl/certs/; mv /etc/puppetlabs/puppet/ssl/pki/cacerts/cacerts.pem /etc/puppetlabs/puppet/ssl/certs/ca.pem')
+    shell('mkdir -p /etc/puppetlabs/puppet/ssl/private_keys; mv /etc/puppetlabs/puppet/ssl/pki/private/centos7-64-1.pem /etc/puppetlabs/puppet/ssl/private_keys/')
+    shell('mv /etc/puppetlabs/puppet/ssl/pki/public/centos7-64-1.pub /etc/puppetlabs/puppet/ssl/certs/centos7-64-1.pem')
+  end
+  context 'default parameters' do
+    # Using puppet_apply as a helper
+    it 'works with no errors' do
+      pp = 'include hyperglass::server; include hyperglass::agent'
```

Comment:
Just wondering, is this valid syntax? I thought it was
```suggestion
      pp = 'include hyperglass::server, hyperglass::agent'
```

Reply:
yes it is valid :)